combat-trainer.lic

```diff
@@ -1488,7 +1488,7 @@ class SpellProcess
     return if @training_spells_max_threshold && game_state.npcs.length > @training_spells_max_threshold
     return if DRStats.mana < @training_spell_mana_threshold
 
-    needs_training = %w[Warding Utility Augmentation Sorcery]
+    needs_training = %w[Warding Utility Augmentation Sorcery Debilitation Targeted\ Magic]
```

Comment:
I suggest we refactor this list to standard looking array of strings to avoid ambiguity about the escaped space `\ `
```suggestion
    needs_training = ['Warding', 'Utility', 'Augmentation', 'Sorcery', 'Debilitation', 'Targeted Magic']
```

Reply:
I was working within the bounds of %w and could not get it to work without the escaped space. Wonder why it was %w now...

Reply:
Someone trying to be cheeky with Ruby syntax, heh

Reply:
Either way, when using %w, you need to escape `\` spaces. That took a bit to find. I will test the other way.

Reply:
`%w[]` is the convention when it's easy to do (Rubocop will auto-fix this). Since we need the escaped space, I'm fine with going to `['...']` syntax

Reply:
Changed to `['..']`. Works as expected.